test/ysh-prettify.sh

```diff
@@ -431,7 +444,7 @@ test-posix-func() {
   proc f
   {
     echo "hi"
-  }'
+  }' \
```

Comment:
does this do anything?  looks a little weird

Reply:
No it doesn't :P. I'll remove that.